lib/puppet/pops/serialization/to_data_converter.rb

```diff
@@ -14,6 +14,8 @@ class ToDataConverter
     # @option options [Boolean] :local_reference use local references instead of duplicating complex entries
     # @option options [Boolean] :type_by_reference `true` if Object types are converted to references rather than embedded.
     # @option options [Boolean] :symbol_as_string `true` if Symbols should be converted to strings (with type loss)
+    # @option options [Boolean] :force_symbol `true` if Symbols should not be converted (rich_data and symbol_as_string must be false)
+    # @option options [Boolean] :silences_warnings `true` if warnings should be silenced
```

Comment:
Nit: these should both be \`false\` as the docs should represent the default values for the options